backend/app/models.py

```diff
@@ -163,11 +163,19 @@ class GEOIDS(BaseModel):
     geoids: list[str]
 
 
+class ZoneAndGEOIDS(GEOIDS):
+    zone: int | None
+
+
 class ShatterResult(BaseModel):
     parents: GEOIDS
     children: list[Assignments]
 
 
+class UnShatterResult(BaseModel):
```

Comment:
PP: Why double nest the result when we could just return `GEOIDS`? I guess it's not a bad idea to be explicit.

Reply:
be3a89f6ed0d48fb1b042faa9ca4601eab2d074e